src/Domain/Amr/Tags/Flags.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Amr/Flag.hpp"
+
+namespace domain::amr::Tags {
+/// domain::amr::Flag%s for an Element.
+template <size_t VolumeDim>
+struct Flags : db::SimpleTag {
+  using type = std::array<domain::amr::Flag, VolumeDim>;
```

Comment:
Add a `name` function to output this as "AmrFlags" in log messages? Same for comment for "NeighborAmrFlags".

Reply:
I think it will be pretty clear from its value what type of flag it is...